les/peer.go

```diff
@@ -937,6 +943,7 @@ func (p *clientPeer) freezeClient() {
 // Handshake executes the les protocol handshake, negotiating version number,
 // network IDs, difficulties, head and genesis blocks.
 func (p *clientPeer) Handshake(td *big.Int, head common.Hash, headNum uint64, genesis common.Hash, server *LesServer) error {
+	p.headInfo = blockInfo{Hash: head, Number: headNum, Td: td}
```

Comment:
Please add a comment for it. 

e.g. The client won't share the local chain info by default. Use the empty headInfo here.

Reply:
Done, now it is commented at both the client, the server and the common handshake.